ref/configs/local/config.yaml

```diff
@@ -0,0 +1,18 @@
+environment: local
+packages:
+  - name: external-secrets
+    namespace: external-secrets
+    install: "true"
+    manifestPath: "external-secrets/manifests"
+#  - name: keycloak
+#    namespace: keycloak
+#    install: "true"
+#    manifestPath: "keycloak/manifests"
```

Comment:
Does manifests path needs to be changed with `helm` as per the keycloak package?